alpha/apps/kaltura/lib/multiLingualUtils.php

```diff
@@ -245,12 +245,25 @@ protected static function getMappingForDefaultLanguage(&$responseObject, &$multi
 		}
 	}
 	
+	protected static function getMappedLanguages($newMultiLingualMapping, $supportedFields)
+	{
+		$mappedLanguages = array();
+		foreach ($supportedFields as $field)
+		{
+			array_push($mappedLanguages, array_keys($newMultiLingualMapping[$field]));
```

Comment:
add comment that explains that if we would a values in a language in at least one of the fields - it is supported and should be part of the supported language map